api/downtime.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Observatory Control System
+show_sidebar: false
+hide_hero: true
+hide_footer: true
+---
+<!--- FIXME: If we go with this theme, fix this link -->
```

Comment:
When I set the url value to `/assets/html/downtime.html`, the url resolves to `https://eheinrich.github.io/assets/html/downtime.html`. I'm pretty sure that when this isn't running on a fork, then the url would be generated correctly? I tried updating both the `baseurl` and `url` settings in `_config.yaml` to get the link to generate correctly when deployed on this fork, didn't work though.